daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/dafext.xsd

```diff
@@ -119,6 +119,20 @@
             </xs:documentation>
           </xs:annotation>
         </xs:element>
+        <xs:element name="outputStreamChunkSizeInBytes" type="xs:int" default="4096" minOccurs="0">
```

Comment:
This is an incredibly small number in this modern age. I would suggest a minimum of 64K.